components/inputs/README.md

```diff
@@ -38,3 +39,42 @@ The `<d2l-input-text>` element is a simple wrapper around the native `<input typ
 - `step` (optional, String): for number inputs, sets the step size
 - `type` (optional, String): can be one of `text` (default), `email`, `number`, `password`, `tel`, `url`
 - `value` (optional, String): value of the input
+
+## Search Inputs
+
+For text searches use `<d2l-input-search>`, which wraps the native `<input type="search">` element.
+
+![example screenshot of search input](./screenshots/search.png?raw=true)
```

Comment:
This screenshot doesn't seem to work for me - looks like the extension here is png but the one added is a gif. The same problem seems to be happening in the `Text Inputs` section.

Reply:
Yep you're right -- fixed, thanks!